Sources/sentry/include/SentryApp.h

```diff
@@ -5,4 +5,28 @@
 
 #include <sentry.h>
 
-#endif
\ No newline at end of file
+#ifdef _WIN32
```

Comment:
Can we define the new code in a separate header like `SwiftFatalErrorMessage.h`?

Reply:
Done